webpack.config.js

```diff
@@ -1,13 +1,30 @@
 module.exports = {
   module: {
-    loaders: [{
-      exclude: /(node_modules)/,
-      loader: 'babel-loader',
-      query: {
-        presets: ['es2015'],
-        plugins: [
-          ['transform-object-rest-spread', { useBuiltIns: true }],
-        ],
+    rules: [{
+      test: /\.js$/,
+      exclude: /node_modules/,
+      use: {
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            [
+              '@babel/preset-env',
+              {
+                corejs: '2.6.9',
```

Comment:
This is interesting. I thought only 2 or 3 were valid values for this and didn't see where you could set it like this. Can you refer me to the docs for this?

Reply:
It looks like you are correct. I do not really see much documentation on setting this property but all the example code on their site seem to point towards a single integer. 

I copied this format from some Github example. I will switch it since I think it makes more sense to use a major version instead of the full semver. 
```suggestion
                corejs: 2,
```